Model/ShimModel.php

```diff
@@ -54,6 +56,7 @@ public function __construct($id = false, $table = null, $ds = null) {
 			'alias' => $this->alias,
 			'schema' => $this->schemaName,
 		];
+		$this->addBehavior('Containable');
```

Comment:
what if it is already in $actAs?
I think these changes will create dangerous side effects.
If you want to do this, you must collect all in $actsAs, optionally add Containable into it and then load them together.
And only if there were no such definitions you can execute this line here.